omnibus/package-scripts/agent/prerm

```diff
@@ -59,12 +64,15 @@ deregister_agent()
     elif [ "$DISTRIBUTION_FAMILY" = "Debian" ]; then
         if command -v update-rc.d >/dev/null 2>&1; then
             update-rc.d -f $SERVICE_NAME-process remove || true
-            update-rc.d -f $SERVICE_NAME-sysprobe remove || true
             update-rc.d -f $SERVICE_NAME-trace remove || true
             update-rc.d -f $SERVICE_NAME remove || true
         else
             echo "[ WARNING ]\tCannot detect a supported init system. The datadog-agent package only provides service files for systemd, upstart and sysvinit."
         fi
+    elif [ "$SUSE_SYSVINIT_SUPPORT"="yes" ]; then
```

Comment:
Not sure if this works without spaces between the operator and operands.

Reply:
Indeed, it only worked in the tests we do because this always returns true.